api/v4/common_types.go

```diff
@@ -387,12 +387,12 @@ type EsDefaults struct {
 	//     strict: Ensure that SSL is enabled
 	//             in the web.conf configuration file to use
 	//             this mode. Otherwise, the installer exists
-	//	     	   with an error.
+	//	     	   with an error. This is the DEFAULT mode used 
+	//             by the operator if left empty.
```

Comment:
We need to run make generate to generate crd yaml changes here?

Reply:
Arjun, I plan to do this after we have `develop` changes in es branch so I have full /complete generated crd yaml file.